### PR TITLE
Add remove button to cells

### PR DESCRIPTION
**Pull request by hbcarlos**
A basic approach for #33 

ezgif com-gif-maker

### CODE REVIEW

**Comment by hbcarlos** [comment]
Change icon to a red trash can.

File: src/editor/components/gridItem.ts [hunk 1]
```diff
@@ -124,23 +133,28 @@ export class GridItem extends Panel {
 
     const content = document.createElement('div');
     content.className = 'grid-stack-item-content';
+
+    const button = document.createElement('div');
+    button.className = 'close-button';
+    const close = document.createElement('div');
```
Comment (jtpio) [inline src/editor/components/gridItem.ts:139]: We might be able to use a `LabIcon` here, to make it easier to handle themes: https://jupyterlab.readthedocs.io/en/latest/developer/ui_components.html#labicon-set-up-and-render-icons

The icon could even be the same as the one used for the `Delete` entry in the file browser context menu:

image

Comment (jtpio) [reply src/editor/components/gridItem.ts:139]: Done in https://github.com/hbcarlos/voila-editor/pull/36/commits/ac90960094f661a097de0ea3c21401910d231524

trash-labicon

**Comment by jtpio** [comment]
Thanks!

**Pull request merged**